bin/shield-linter.js

```diff
@@ -62,7 +62,7 @@ function _loadConfig(cfgPath, cfgRulesPath) {
   try {
     const rules = require(cfgRulesPath);
     const cfg = require(path.resolve(cfgPath));
-    Object.values(rules).forEach(rule => rule(cfg));
+    Object.values(rules).forEach(rule => rule(cfg, cfgPath));
```

Comment:
I needed to pass the path to the config file (manifest.json or package.json) because I need to resolve the relative paths in the manifest.json's `experiment_apis` object keys.